unic/ucd/category/src/category.rs

```diff
@@ -80,7 +80,36 @@ pub enum GeneralCategory {
     Unassigned,
 }
 
-use self::GeneralCategory::*;
+use GeneralCategory::UppercaseLetter as Lu;
```

Comment:
No, I was just doing the minimal required in order to get the new table to compile, in full expectation that the `char_property!` macro is coming soon, to replace this.

Though, since it's like three lines away from being the `abbr_names` module, I may as well do that, I now realize.

(I'll also modify the Python script to generate this changed table as well)